tests/httpClient.test.js

```diff
@@ -1813,7 +1835,7 @@ tap.test('httpClient', test => {
         assert.match(beforeRequest, {
           level: 10,
           msg: /^make call$/,
-          url: new RegExp(`^${MY_AWESOME_SERVICE_PROXY_HTTP_URL}/foo$`),
+          url: '/foo',
```

Comment:
Why `/foo` without the MY_AWESOME_SERVICE_PROXY_HTTP_URL?

Reply:
Because the url is what you receive from an instantiated proxy.

I've added logs for `baseURL`